app/main/controller/presto_controller.py

```diff
@@ -0,0 +1,23 @@
+from flask import request, current_app as app
+from flask_restplus import Resource, Namespace
+
+from app.main.lib.fields import JsonObject
+from app.main.lib import similarity
+
+api = Namespace('presto', description='presto callback url')
+presto = api.model('presto', {
+    'body': JsonObject(required=False, description='Original body of the message sent - will contain ID, callback_url, and original url/text of request'),
+    'response': JsonObject(required=False, description='The response data sent back from the model'),
+})
+@api.route('/receive/<string:action>/<string:model_type>')
+class PrestoResource(Resource):
```

Comment:
maybe this could be 'callback_controller', since other services (like timpani) would use this also?

Reply:
See my other comment w/r/t service-based callbacks

Reply:
I had the same question. Will look for the other comment :-)